pkg/controller/networkpolicy/convert.go

```diff
@@ -37,23 +37,9 @@ func ConvertClusterGroupCRD(Object *unstructured.Unstructured, toVersion string)
 		return nil, statusErrorWithMessage("conversion from a version to itself should not call the webhook: %s", toVersion)
 	}
 	switch Object.GetAPIVersion() {
-	case "crd.antrea.io/v1alpha2":
-		switch toVersion {
-		case "crd.antrea.io/v1alpha3", "crd.antrea.io/v1beta1":
-			ipb, found, err := unstructured.NestedMap(convertedObject.Object, "spec", "ipBlock")
-			if err == nil && found && len(ipb) > 0 {
-				unstructured.RemoveNestedField(convertedObject.Object, "spec", "ipBlock")
-				// unstructured.SetNestedSlice expects a slice of interface as value
-				ipBlocks := make([]interface{}, 1)
-				ipBlocks[0] = ipb
-				unstructured.SetNestedSlice(convertedObject.Object, ipBlocks, "spec", "ipBlocks")
-			}
-		default:
-			return nil, statusErrorWithMessage("unexpected conversion version %q", toVersion)
-		}
 	case "crd.antrea.io/v1alpha3", "crd.antrea.io/v1beta1":
 		switch toVersion {
-		case "crd.antrea.io/v1alpha2", "crd.antrea.io/v1alpha3", "crd.antrea.io/v1beta1":
+		case "crd.antrea.io/v1alpha3", "crd.antrea.io/v1beta1":
```

Comment:
Based on this change, it seems that we no longer need the conversion webhook at all for this CRD. IIRC the schema is the same for the `v1alpha3` and `v1beta1` versions. If this is true, we could remove this code and change the conversion strategy from `Webhook` to `None` in the CRD definition.

Adding @Dyanngg and @tnqn in case I am misremembering something.

Reply:
Yes that is correct. We do not need custom conversion for `v1alpha3` <-> `v1beta1`

Reply:
Removed the conversion webhook. Thanks.